scripts/scenarios/cloud8/qa/no-ssl/qa-scenario-3.yaml

```diff
@@ -10,7 +10,6 @@ proposals:
         export: ##shared_nfs_export_for_glance##
         mount_path: "/images"
         mount_options:
-        - nfsvers=3
```

Comment:
Can we remove also the attribute key `mount_options`, having it there empty will be confusing. The same is the case in 6a scenario.

Reply:
Fixed